src/Kobo/Model.cpp

```diff
@@ -96,3 +97,17 @@ GetKoboWifiInterface() noexcept
 {
   return DetectKoboModel() == KoboModel::LIBRA2 ? "wlan0" : "eth0";
 }
+
+const char* 
+GetKoboModel() noexcept
+{
+  static char buffer[16];
+  if (!ReadFromFile("/dev/mmcblk0", 0x200, buffer, sizeof(buffer)))
+    buffer[0]=0 ;
+  
+  // trunc the string to 8 char
+  buffer[8]=0 ;
+
+  return buffer ;
+
+}
```

Comment:
Unused (and undocumented) code.

Reply:
I thought it was pushed on my branch
The code is used for debug in the system window.